src/target/linux/mod.rs

```diff
@@ -38,6 +41,11 @@ impl LinuxTarget {
         ReadMemory::new(self.pid())
```

Comment:
Should `ReadMemory` also be moved to a new file?

Reply:
Definitely - will do in a next commit :)